sdk/client-core/core/src/main/java/io/clientcore/core/http/pipeline/HttpLoggingPolicy.java

```diff
@@ -344,12 +335,10 @@ private static long getContentLength(ClientLogger logger, HttpHeaders headers) {
      *
      * @param contentTypeHeader Content-Type header value.
      * @param contentLength Content-Length header represented as a numeric.
-     *
      * @return A flag indicating if the request or response body should be logged.
      */
     private static boolean shouldBodyBeLogged(String contentTypeHeader, long contentLength) {
-        return !APPLICATION_OCTET_STREAM.equalsIgnoreCase(contentTypeHeader)
-            && contentLength != 0
+        return !APPLICATION_OCTET_STREAM.equalsIgnoreCase(contentTypeHeader) && contentLength != 0
```

Comment:
Does [this comment ](https://github.com/Azure/azure-sdk-for-java/pull/39093#discussion_r1535967042) apply here?

Reply:
It does, mind filing an issue for it as this PR has a lot of merge conflict spots